contrib/inventory/foreman.py

```diff
@@ -46,6 +46,13 @@
 
 from requests.auth import HTTPBasicAuth
 
+try:
+    import concurrent.futures
```

Comment:
My main concern with this is that `futures` is not available in base RHEL, Satellite, or Ansible Engine subscription channels for Red Hat Customers. The only channels I see it currently available in is OpenStack for EL6 and both OpenStack and Gluster for EL7, both which are add-on SKUs and not available to customers who haven't bought those other products..

From a community upstream standpoint, this is perfectly fine. However, if the intent is for this to be used with the product version of Satellite (which I assume it is given the use of the product name in the PR) then that's problematic.

Reply:
also, this script is on it's way out, use the foreman inventory plugin instead